src/parsing/lpParser.mly

```diff
@@ -90,6 +90,7 @@
 
 %token <bool * string> DEBUG_FLAGS
 %token <string> INT
+%token <Path.t * string> PINT
```

Comment:
Instead of introducing a new token, I suggest to change the type of INT by adding a path.